src/utils/apiTypes.ts

```diff
@@ -16,5 +18,6 @@ export type GetQuestionsResponse = {
 
 export type Unit = {
   id: number,
-  name: string
+  name: string,
+  type: UnitType
```

Comment:
Maybe make type UnitType | undefine since it is not needed when we return all units.